src/lib/apirequest.ts

```diff
@@ -135,49 +147,62 @@ export function createAPIRequest(parameters, callback) {
   if (parameters.mediaUrl && media.body) {
     options.url = parameters.mediaUrl;
     if (resource) {
+      // Axios doesn't support multipart/related uploads, so it has to
+      // be implemented here.
       params.uploadType = 'multipart';
-      options.multipart = [
+      const multipart = [
         {'Content-Type': 'application/json', body: JSON.stringify(resource)}, {
           'Content-Type':
               media.mimeType || (resource && resource.mimeType) || defaultMime,
           body: media.body  // can be a readable stream or raw string!
         }
       ];
+      const boundary = uuid.v4();
+      const finale = `--${boundary}--`;
+      const rStream = new stream.PassThrough();
+      const isStream = isReadableStream(multipart[1].body);
+      headers['Content-Type'] = `multipart/related; boundary=${boundary}`;
```

Comment:
It seems like we need a standalone module that provides multipart support on top of axios. [form-data](https://www.npmjs.com/package/form-data) might be relevant.

Reply:
I filed an issue against the axios repo:
https://github.com/axios/axios/issues/1247

Based on their feedback, I'll either submit a PR or go build a standalone module.

Reply:
possibly relevant: https://github.com/axios/axios/issues/789